research/mfac/tutorials/one_shot_pruning_with_mfac.md

```diff
@@ -118,11 +131,12 @@ data_loader = DataLoader(dataset, shuffle=True, batch_size=16)
 # wrap data loader to send data to correct device 
 def device_data_loader():
     for sample in data_loader:
-        yield [t.to(device) if isinstance(t, torch.Tensor) else t for t in sample]
+        img, target = [t.to(device) if isinstance(t, torch.Tensor) else t for t in sample]
+        yield [img], {}, target
 
 # wrap cross entropy loss function to use correct output
-def loss_function(x, y):
-    return torch.nn.functional.cross_entropy(x[0], y)
+def loss_function(model_outputs, loss_target):
+    return torch.nn.functional.cross_entropy(model_outputs[0], loss_target)
```

Comment:
shouldn't we have an assert here that model_outputs is a tuple? Also worth adding a check that the shapes of loss target and model outputs match right?